test/integration/replicaset/test_replicaset_upgrade.py

```diff
@@ -0,0 +1,245 @@
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+from time import sleep
+
+import pytest
+from replicaset_helpers import stop_application
+from vshard_cluster import VshardCluster
+
+from utils import get_tarantool_version, run_command_and_get_output, wait_file
+
+tarantool_major_version, _ = get_tarantool_version()
+
+
+def run_command_on_instance(tt_cmd, tmpdir, full_inst_name, cmd):
+    con_cmd = [tt_cmd, "connect", full_inst_name, "-f", "-"]
+    instance_process = subprocess.Popen(
+        con_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        stdin=subprocess.PIPE,
+        text=True,
+    )
+    instance_process.stdin.writelines([cmd])
+    instance_process.stdin.close()
+    output = instance_process.stdout.read()
+    return output
+
+
+@pytest.mark.skipif(
+    tarantool_major_version < 3, reason="skip centralized config test for Tarantool < 3"
+)
```

Comment:
The tests in the file is not about centralized config, why do we need to skip it?

Reply:
Rewrited skip condition according to https://github.com/tarantool/tt/pull/936#discussion_r1818115933